dali/python/nvidia/dali/triton.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import inspect
+
+
+def _discover_autoserialize(module, visited=[]):  # TODO test
+    """
+    TODO
+    :param module:
+    :return:
+    """
+    assert module is not None
+    ret = []
+    mems = inspect.getmembers(module)
+    modules = []
+    for mem in mems:
+        obj = getattr(module, mem[0], None)
+        if inspect.ismodule(obj) and mem[1] not in visited:
+            modules.append(mem[0])
+            visited.append(mem[1])
+        elif inspect.isfunction(obj) and getattr(obj, 'autoserialize_me', False):
```

Comment:
For clarity, maybe wrap the check `getattr(obj, 'autoserialize_me', False)` into function like `is_marked_autoserializable` or something?

Reply:
Done